spec/features/admin/admin_cannot_modify_users_spec.rb

```diff
@@ -1,3 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin can modify their account data' do
+  scenario 'Admin can not modify user data' do
+    admin = create(:user, role: 1)
+    user = create(:user)
+
+    visit login_path
+
+    fill_in 'Username', with: admin.username
+    fill_in 'Password', with: admin.password
+
+    click_on 'Save Session'
+
+    expect(page).to have_content('Modify Account Data')
+
+    click_on 'Modify Account Data'
+    expect(current_path).to eq(edit_user_path(admin))
```

Comment:
@brendandillon Definitely would be useful I agree. So I just updated the test, but the admin is not receiving a 404 when they visit the edit users page. I think we need to take a look at the authorization.